packages/common/src/audius-query/createApi.ts

```diff
@@ -272,7 +272,7 @@ const fetchData = async <Args, Data>(
     endpoint.onQueryStarted?.(fetchArgs, { dispatch })
 
     const apiData = await endpoint.fetch(fetchArgs, context)
-    if (!apiData) {
+    if (apiData == null) {
```

Comment:
This line was breaking fetch endpoints which return falsey values, most notably `count` endpoints when they return 0.